Model/NavigationConfig.php

```diff
@@ -126,7 +128,10 @@ public function getJsFormConfig()
                 'productListSelector' => '.products.wrapper',
                 'toolbarSelector' => '.toolbar.toolbar-products',
                 'ajaxCache' => true,
-                'urlStrategy' => $this->config->getUrlStrategy() === 'Tweakwise\Magento2Tweakwise\Model\Catalog\Layer\Url\Strategy\QueryParameterStrategy' ? 'queryparameter' : 'path',
+                'urlStrategy' => $this->config->getUrlStrategy() ===
+                self::QUERY_PARAMETER_STRATEGY
```

Comment:
Use the following:

`$this->config->getUrlStrategy() === QueryParameterStrategy::class`

See `Model/Config.php:292`

Reply:
I cant, because then i need to compare it to the full classname and then line 133 gets to long

Reply:
Implemented changes